api/core/workflow/utils/structured_output/entities.py

```diff
@@ -0,0 +1,16 @@
+from enum import StrEnum
+
+
+class ResponseFormat(StrEnum):
```

Comment:
I suggest enhancing the documentation for the `ResponseFormat` enum.  

For instance, provide a brief description for each enum value, highlighting their distinctions, and include examples to illustrate their usage.